drf_spectacular/utils.py

```diff
@@ -62,20 +62,71 @@ class OpenApiSchemaBase:
     pass
 
 
+class OpenApiExample(OpenApiSchemaBase):
+    """
+    https://swagger.io/specification/
+     * Specification
+        * Schema
+           * ExampleObject
+    """
+    summary: str
+    description: str
+    value: Any
+
+    # below attributes is not openapiSpec Object, only for drf-spectacular
+    request_only: bool
+    response_only: bool
+
+    def __init__(
+            self,
+            summary="",
```

Comment:
So i gave this another detailed look. i think 2 things would improve the inteface.

1. for consistency with `OpenApiParameter` i would change the interface slightly by pulling in the example name as mandatory parameter. that also means all example listings are lists and not dicts. 
2. `name` first, `value` second. no defaults because the example makes no sense without name and value

```python
    def __init__(
            self,
            name: str,
            value: Any
            summary="",
            ...
)
```

so that `@extend_schema(examples=[OpenApiParameter('ex1', value_dict), ...]`

Reply:
it's more elegant solution